src/tss2-tcti/tctildr-nodl.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ * Copyright 2017-2018, Fraunhofer SIT sponsored by Infineon Technologies AG
+ * Copyright 2019, Intel Corporation
+ * All rights reserved.
+ */
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include <stdlib.h>
+#include <errno.h>
+#include <string.h>
+#include <inttypes.h>
+
+#include "tctildr.h"
+#include "tss2_tcti_mssim.h"
+#ifdef _WIN32
+#include "tss2_tcti_tbs.h"
+#else /* _WIN32 */
+#include "tss2_tcti_device.h"
+#endif
+#define LOGMODULE tcti
+#include "util/log.h"
+
+#define ARRAY_SIZE(X) (sizeof(X)/sizeof(X[0]))
+
+struct {
+    TSS2_TCTI_INIT_FUNC init;
+    char *conf;
+    char *description;
+} tctis [] = {
+#ifdef _WIN32
+    {
+        .init = Tss2_Tcti_Tbs_Init,
+        .description = "Access to TBS",
+    },
+#else
+#ifdef TCTI_DEVICE
+    {
+        .init = Tss2_Tcti_Device_Init,
+        .conf = "/dev/tpmrm0",
+        .description = "Access to /dev/tpmrm0",
+    },
+    {
+        .init = Tss2_Tcti_Device_Init,
+        .conf = "/dev/tpm0",
+        .description = "Access to /dev/tpm0",
+    },
+#endif /* TCTI_DEVICE */
```

Comment:
VxWorks will also need an #ifdef since they export the device under "/tpm0".
See https://github.com/tpm2-software/tpm2-tss/pull/1439/files#diff-e5fc82fbafb52375569e9a8a6c9d603cR67-R69

Reply:
> is [9f1bdd1](https://github.com/tpm2-software/tpm2-tss/commit/9f1bdd187c7dbcd1bf4e439f9cf52962483c7468) even needed, if [8d973ab](https://github.com/tpm2-software/tpm2-tss/commit/8d973abcb3fcf446d467861ba918703e5e2152b5) removes the file anyways ?

9f1bdd1 was added to make 8d973ab possible. Couldn't get it to build without that change. If you'd like me to squash them together I'm not against the idea.

Reply:
> VxWorks will also need an #ifdef since they export the device under "/tpm0".
> See https://github.com/tpm2-software/tpm2-tss/pull/1439/files#diff-e5fc82fbafb52375569e9a8a6c9d603cR67-R69

That PR hasn't been merged. Are you asking me to do something specific here?

Reply:
Well, the other PR won't merge ontop of this one and this one will not merge / revert the other one...
So I don't know what to do.
I can merge the other and you rebase and include here ?
Sorry for the inconvenience...

Reply:
I hadn't been tracking #1433. Saw it get closed, didn't see it reappear in #1439. Since you're asking for input here it seems to me you have reservations about merging #1439? Would you like me to do a review of this PR?

Reply:
Ì just re-posted my review with some "hint"...